UsingInPipelines.md

```diff
@@ -0,0 +1,64 @@
+# Examples of Pipelines
+
+## GitHub Actions
+
+```yaml
+name: Build
+
+on:
+  workflow_dispatch:
+  push: 
+    branches:
+      - main
+    paths:
+      - 'src/**'
+      - '.github/workflows/build.yaml'
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container:
+      image: m365pnp/spfx:1.14.0
+    steps:
+      - uses: actions/checkout@v2
+      - run: npm install
+      - run: gulp bundle --ship
+```
+
+Reference run: https://github.com/alexsapozhkov/codespaces-test-04/runs/7358446857?check_suite_focus=true
```

Comment:
Let's mark the URL as a hyperlink for easier navigation

Reply:
I think I removed UsingInPipelines.md document for the time being